server/models/reviews.js

```diff
@@ -1,10 +1,22 @@
 const axios = require('axios');
 
-const endpoint = process.env.BASEURL + '/reviews';
+const endpoint = process.env.BASEURL + '/reviews/';
```

Comment:
```suggestion
const endpoint = process.env.BASEURL + '/reviews';
```
reason: not all of these api requests will be /reviews/, but all these api requests will start with /reviews

Reply:
Understood, thank you for the clarification!